alerts/apps_alerts.libsonnet

```diff
@@ -156,7 +156,7 @@
               severity: 'warning',
             },
             annotations: {
-              message: 'Job {{ $labels.namespaces }}/{{ $labels.job }} is taking more than 1h to complete.',
+              message: 'Job {{ $labels.namespaces }}/{{ $labels.exported_job }} is taking more than 1h to complete.',
```

Comment:
In kube-state-metrics 1.4 this label has actually been renamed due to the collision to `job_name`.